src/types/plugin-input.ts

```diff
@@ -0,0 +1,35 @@
+import { SupportedEvents, SupportedEventsU } from "./context";
+import { StaticDecode, Type as T } from "@sinclair/typebox";
+import { StandardValidator } from "typebox-validators";
+
+export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU extends SupportedEvents[T] = SupportedEvents[T]> {
+  stateId: string;
+  eventName: T;
+  eventPayload: TU["payload"];
+  settings: StartStopSettings;
+  authToken: string;
+  ref: string;
+}
+
+const ONE_DAY = 24 * 60 * 60 * 1000;
+
+export const startStopSchema = T.Object({
+  isEnabled: T.Boolean(),
```

Comment:
```suggestion
  isEnabled: T.Boolean({ default: true} }),
```

I think it makes more sense to enable it by default.